src/recipe/recipe.dao.ts

```diff
@@ -50,16 +53,27 @@ export class RecipeDAO {
         },
       });
     }
+
+    if (newRecipe.image && recipeToUpdate.image) {
+      await this.prismaService.image.delete({
+        where: {
+          recipeId,
+        },
+      });
+    }
+
     const updatedRecipe = await this.prismaService.recipe.update({
       where: { id: recipeId },
       data: {
         ...newRecipe,
         preparing: { create: newRecipe.preparing },
         ingredients: { create: newRecipe.ingredients },
+        image: { create: newRecipe.image },
```

Comment:
Why do you create create "image" whiting recipe update process? Is image upload the part of upload recipe endpoint?

Reply:
no it isn not, but i use updateRecipe method while attaching image to recipe in uploadRecipeImage handler